src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -66,20 +67,24 @@ export default class MxGraphRenderer {
       const bounds = shape.bounds;
       const parent = this.getParent(bpmnElement);
       const absoluteCoordinate = { x: bounds.x, y: bounds.y };
-      let style: string;
-      if (bpmnElement instanceof ShapeBpmnEvent) {
-        console.log('_________', 'ShapeBpmnEvent', bpmnElement);
-        // TODO: following if is just temporary as long as Start Event (subtypes) support is not added
-        if (bpmnElement.kind === ShapeBpmnElementKind.EVENT_END) {
-          style = bpmnElement.eventKind && bpmnElement.eventKind !== 'none' ? bpmnElement.kind + '_' + bpmnElement.eventKind : bpmnElement.kind;
-        } else {
-          style = bpmnElement.kind;
-        }
+      const style: string = this.getStyleName(bpmnElement);
```

Comment:
:warning: if the function declared a string returned type, type declaration won't be needed here

Reply:
yes definitely - I haven't noticed

Reply:
done